### DISCUSSION TITLE
Where did the Dreambooth Input tab disappear?

### DISCUSSION THREAD

**Discussion by boantara** [d8ahazard/sd_dreambooth_extension#1454] [Q&A]
As you can see in the image, the input tab is not visible. Does anyone know the solution?

![image](https://github.com/d8ahazard/sd_dreambooth_extension/assets/43796869/40a31293-c071-464e-b377-672773fa3ff4)

**Comment by swathi-kumar-github** [comment]
Same issue:( 
Did you find any solution?

**Comment by envl2** [comment]
Im pretty sure they've removed the input tab at some point and replaced it with the parameters tab which has some of the functionality of the input tab (some functionality missing like the "Performance Wizard"). The same thing goes for the concepts tab. You can also use an older version of the extension. This version seems to work for me "https://github.com/d8ahazard/sd_dreambooth_extension/releases/tag/1.0.1"


![abcdefghijklmnopqrstuvwxyz](https://github.com/user-attachments/assets/e42b0af9-e78c-4220-8e7c-b0d7cc1391fe)
